ecosystem/models/test_results.py

```diff
@@ -1,10 +1,18 @@
 """Test results for commands."""
 import datetime
 from typing import Optional, List
+from dataclasses import dataclass
 
 from .utils import JsonSerializable
 
 
+@dataclass
+class TestStatus:
```

Comment:
This should probably be an enum?

Reply:
Thanks! Fixed in [04ef375](https://github.com/qiskit-community/ecosystem/pull/457/commits/04ef3755d8e6156c52f9f98b5b7072c5663afd0c)